README.md

```diff
@@ -1,99 +1,66 @@
-# Tellor
+# Tellor Pallet
 
 [![Check Set-Up & Build](https://github.com/tellor-io/tellor-pallet/actions/workflows/check.yml/badge.svg?branch=main)](https://github.com/tellor-io/tellor-pallet/actions/workflows/check.yml)
 [![Run Tests](https://github.com/tellor-io/tellor-pallet/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/tellor-io/tellor-pallet/actions/workflows/test.yml)
 
-This repo is a clone of this work [here](https://github.com/evilrobot-01/substrate-pallets/tree/tellor).
+## Overview
 
-## Porting Progress
+This pallet introduces Tellor oracle functionality to parachains, controlled by Tellor staking and governance smart contracts on another EVM smart contract enabled parachain.
 
-### AutoPay
+A parachain first registers itself with the controller contracts. After that, anyone can request oracle data to be reported to the parachain by either creating a 'tip' for a onetime report, or by creating and funding a feed for recurring reports. 
+Reporters are required to first stake Tellor Tributes (TRB) into the Tellor parachain staking contract before they can begin reporting directly to the parachain.
 
-| Dispatchable Function                  |  Functional Test   | Notes |
-|----------------------------------------|:------------------:|-------|
-| :white_check_mark: `claim_onetime_tip` | :white_check_mark: |       |
-| :white_check_mark: `claim_tip`         | :white_check_mark: |       |
-| :white_check_mark: `fund_feed`         | :white_check_mark: |       |
-| :white_check_mark: `setup_data_feed`   | :white_check_mark: |       |
-| :white_check_mark: `tip`               | :white_check_mark: |       |
+Anyone can then dispute a reported value, provided they lock a dispute fee. 
+Votes are collated and sent to the Tellor parachain governance contract for tallying and execution.
+A successful dispute results in a disputed reporter being slashed and the slash amount being awarded to the dispute initiator. 
+The dispute fee is awarded to the disputed reporter if the dispute is unsuccessful. 
```

Comment:
I'd update to specify that fee is awarded to reporter if dispute fails and fee is given back to initiator if tallied and executed as invalid. [src](https://github.com/tellor-io/governance/blob/main/contracts/Governance.sol#L224)

Reply:
Updated to "The dispute fee is awarded to the disputed reporter if the dispute is unsuccessful, but given back to initiator if tallied and executed as an invalid dispute."